HunterPie/Internal/Initializers/CustomThemeInitializer.cs

```diff
@@ -13,6 +13,14 @@ internal class CustomThemeInitializer : IInitializer
         public void Init()
         {
             string themePath = Path.Combine(ClientInfo.ThemesPath, ClientConfig.Config.Client.Theme);
+            
+            if (!File.Exists(themePath))
+                {
+                    Log.Error("Failed to load theme {0}", ClientConfig.Config.Client.Theme.Current);
+                    Log.Info("Failed to find theme {0}, Changed to Default theme", ClientConfig.Config.Client.Theme.Current);
+                    themePath = Path.Combine(ClientInfo.ThemesPath, "Default");
+                    ClientConfig.Config.Client.Theme.Current = "Default";
+                }
```

Comment:
Just fix the brackets indentation to match the rest of the project, please

Reply:
Fixed :)